discojs/src/models/gpt/layers.ts

```diff
@@ -78,26 +95,35 @@ class CausalSelfAttention extends tf.layers.Layer {
   }
 
   override build (): void {
+    // key, query, value projections for all heads, but in a batch
     this.cAttnKernel = this.addWeight(
-      'c_attn/kernel',
+      'c_attn.weight',
       [this.nEmbd, 3 * this.nEmbd],
       'float32',
-      tf.initializers.glorotNormal({})
+      tf.initializers.randomNormal({ mean:0, stddev:0.02, seed: this.seed }) // use same init as GPT2
     )
     this.cAttnBias = this.addWeight(
-      'c_attn/bias',
+      'c_attn.bias',
       [3 * this.nEmbd],
       'float32',
       tf.initializers.zeros()
     )
+    // output projection
     this.cProjKernel = this.addWeight(
-      'c_proj/kernel',
+      'c_proj.kernel',
       [this.nEmbd, this.nEmbd],
       'float32',
-      tf.initializers.glorotNormal({})
+      // the input keeps accumulating through the residual stream so we
+      // scale the initialization with the nb of layers to keep a unit std
+      // Sources:
+      // https://github.com/karpathy/build-nanogpt/blob/6104ab1b53920f6e2159749676073ff7d815c1fa/train_gpt2.py#L103
+      // https://youtu.be/l8pRSuU81PU?si=5GcKfi_kPgLgvtg2&t=4640
```

Comment:
nope to youtube tracking id

```suggestion
      // https://youtu.be/l8pRSuU81PU?t=4640
```